### DISCUSSION TITLE
How can I apply below bands to a filter?

### DISCUSSION THREAD

**Discussion by tdsoft** [alnitak/flutter_soloud#88] [General]
![image](https://github.com/alnitak/flutter_soloud/assets/5484199/971e9972-8afa-4876-8a83-1f735254285b)
in eqfilter we have only 8 bands and I have 11 bands in this. Is there a way to do this or it is not possible do that in this library?

**Comment by alnitak** [comment]
Hi, I am not familiar with the math needed to compute frequencies for the equalizer, but maybe you can look at [soloud_eqfilter.cpp](https://github.com/alnitak/flutter_soloud/blob/main/src/soloud/src/filter/soloud_eqfilter.cpp) where the eq filter is implemented.

It would be nice to set user parameters for this filter, like the number of bands in a given frequency range. If you will play with this, please let me know!

**Comment by alnitak** [comment]
Looking back to the discussion, I found this that needs an update.

There is now the `parametricEqFilter` that lets you choose the number of bands (1~64) and the window size to compute FFT (a power of 2 from 32 to 4096).
